screenpy_requests/questions/body_of_the_last_response.py

```diff
@@ -1,18 +1,18 @@
-"""
-Investigate the body of the last API response received by the Actor.
-"""
+"""Investigate the body of the last API response received by the Actor."""
 
 from __future__ import annotations
 
 from json.decoder import JSONDecodeError
-from typing import Union
+from typing import TYPE_CHECKING, Union
 
-from screenpy import Actor
 from screenpy.exceptions import UnableToAnswer
 from screenpy.pacing import beat
 
 from ..abilities import MakeAPIRequests
 
+if TYPE_CHECKING:
+    from screenpy import Actor
+
 subscripts = Union[str, int, slice]
```

Comment:
We can move this (and the `from typing import Union`) into the `if TYPE_CHECKING` block.

Reply:
Moved the alias. I let ruff automatically handle the imports.  
I noticed it doesn't bother moving the union down into the TYPE_CHECKING block when other items are being imported from the module.  My guess is it doesn't save anything.

Reply:
Yeah, the main point of moving project code for type hinting into the `if TYPE_CHECKING` block is to reduce the chance of circular imports caused by type hints, and importing from `typing` will never cause that. 

But i kinda like just moving _all_ the type hinting stuff into that block, since we're using it anyway. I dunno, personal preference.